Source/ORTS.Common/SettingsBase.cs

```diff
@@ -43,7 +43,15 @@ protected enum Source
         /// <summary>The store of the settings</summary>
         protected SettingsStore SettingStore { get; private set; }
         /// <summary>Translates name of a setting to its source</summary>
-        protected readonly Dictionary<string, Source> Sources = new Dictionary<string, Source>();
+        protected readonly IDictionary<string, Source> Sources = new Dictionary<string, Source>();
+        /// <summary>
+        /// Translates name of a setting to its value when first loaded.
+        /// </summary>
+        protected readonly IDictionary<string, object> InitialValues = new Dictionary<string, object>();
```

Comment:
The save system (`SettingsBase.Save` IIRC) already handles the prospect of saving a setting with a value equal to its default (it removes it), so this `InitialValues` might be redundant?